### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by abdulhamiid**
- I Created a new game with the name by using the API.
- I Implemented the "Refresh" button which will receiving data from the API and display them.
- I Implemented the form "Submit" button to sending data to the API.
- I Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/modules/async.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import fetchData from './fetch.js';
+
+async function displayScore() {
```
Comment (omar-labana) [inline src/modules/async.js]: - [ ] Kindly, ensure the usage of arrow functions instead of the function keyword

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @abdulhamiid, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/async.js (hunk 1 above)
Comment (abdulhamiid) [reply src/modules/async.js]: okay, thanks

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @abdulhamiid 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abdulhamiid** [comment]
Yeah, great!

**Pull request merged**